mobile/lib/presentation/widgets/people/person_edit_name_modal.widget.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/domain/models/person.model.dart';
+import 'package:immich_mobile/extensions/build_context_extensions.dart';
+import 'package:immich_mobile/extensions/translate_extensions.dart';
+import 'package:immich_mobile/providers/infrastructure/people.provider.dart';
+import 'package:immich_mobile/widgets/common/immich_toast.dart';
+
+class DriftPersonNameEditForm extends ConsumerStatefulWidget {
+  final DriftPerson person;
+
+  const DriftPersonNameEditForm({super.key, required this.person});
+
+  @override
+  ConsumerState<DriftPersonNameEditForm> createState() => _DriftPersonNameEditFormState();
+}
+
+class _DriftPersonNameEditFormState extends ConsumerState<DriftPersonNameEditForm> {
+  late TextEditingController _formController;
+
+  @override
+  void initState() {
+    super.initState();
+    _formController = TextEditingController(text: widget.person.name);
+  }
+
+  void onEdit(String personId, String newName) async {
+    try {
+      final result = await ref.read(driftPeopleServiceProvider).updateName(personId, newName);
+      if (result != 0) {
+        context.pop<String>(newName);
+      }
+    } catch (error) {
+      debugPrint('Error updating name: $error');
+
+      if (!context.mounted) {
+        return;
+      }
+
```

Comment:
The old provider had an invalidate all to refresh the getall provider. We need it here since we've inlined them now. This is also needed in the other modal where the provider was inlined

```suggestion
ref.invalidate(driftGetAllPeopleProvider);
```